app/views/search_form.php

```diff
@@ -127,6 +127,18 @@ class="mainsearch_default_checkbox"
 
                 <fieldset id="advanced_search">
                     <label>Advanced Search options</label>
+                    <a href="#" id="tooltip_search"><span>i</span></a>
+                    <div class="tooltip" id="tooltip_search_text">
+                        <a href="#" id="tooltip_search_close" data-tooltip-id="tooltip_search_text" title="Close this panel">x</a>
```

Comment:
nit: “×” character (U+00D7) vs. x (U+0078) for consistency with the clear button in the search field. If we’re switching characters, we will need to increase font-size